src/components/Statistic/Statistic.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Statistic = (props) => {
```

Comment:
While I think this works, in the other places that we have defined Stateless Components we have used `function name () {` rather than using short syntax. Either is fine, but we should be consistent in our approach. If we leave it as short syntax we can lose the brackets around the single parameter.

Reply:
on it